File: src/Movie/reclamation.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Movie;

import java.sql.Date;
import java.time.LocalDate;

/**
 *
 * @author PersoPc
 */
public class reclamation {
    int id;
    String Titre;
    String Description;
    String traiter;
    Date   datere;

    

    public reclamation(int id, String Titre, String Description, String traiter, Date datere) {
         this.id = id;
        this.Titre = Titre;
        this.Description = Description;
        this.traiter = traiter;
        this.datere = datere;
    }

    reclamation(String Titre, String Description,Date datere) {
         this.Titre = Titre;
        this.Description = Description;
        this.datere = datere;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitre() {
        return Titre;
    }

    public void setTitre(String Titre) {
        this.Titre = Titre;
    }

    public String getDescription() {
        return Description;
    }

    public void setDescription(String Description) {
        this.Description = Description;
    }

    public String getTraiter() {
        return traiter;
    }

    public void setTraiter(String traiter) {
        this.traiter = traiter;
    }

    public Date getDatere() {
        return datere;
    }

    public void setDatere(Date datere) {
        this.datere = datere;
    }
    
    
    
    
    
    
    
    
    
    
    
    
    
}

  

File: Artlife_WEB/src/Entity/Reclamation.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Reclamation
 *
 * @ORM\Table(name="reclamation")
 * @ORM\Entity
 */
class Reclamation
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="Titre", type="string", length=255, nullable=false)
     */
    private $titre;

    /**
     * @var string
     *
     * @ORM\Column(name="Description", type="string", length=255, nullable=false)
     */
    private $description;

    /**
     * @var string
     *
     * @ORM\Column(name="traiter", type="string", length=255, nullable=false)
     */
    private $traiter;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="datere", type="date", nullable=false)
     */
    private $datere;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getTitre(): ?string
    {
        return $this->titre;
    }

    public function setTitre(string $titre): self
    {
        $this->titre = $titre;

        return $this;
    }

    public function getDescription(): ?string
    {
        return $this->description;
    }

    public function setDescription(string $description): self
    {
        $this->description = $description;

        return $this;
    }

    public function getTraiter(): ?string
    {
        return $this->traiter;
    }

    public function setTraiter(string $traiter): self
    {
        $this->traiter = $traiter;

        return $this;
    }

    public function getDatere(): ?\DateTimeInterface
    {
        return $this->datere;
    }

    public function setDatere(\DateTimeInterface $datere): self
    {
        $this->datere = $datere;

        return $this;
    }


}
